libs/dal/src/repositories/base-repository.ts

```diff
@@ -83,7 +83,11 @@ export class BaseRepository<T_DBModel, T_MappedEntity, T_Enforcement = object> {
   }
 
   private calcExpireDate(modelName: string, data: FilterQuery<T_DBModel> & T_Enforcement) {
-    const now: number = Date.now();
+    let now: Date = new Date();
```

Comment:
Maybe rename this to `startDate`? When we override it with expiredAt the "now" might be a bit misleading :)